### PR TITLE
IDLE doc: improve Startup failure subsection.

### PR DESCRIPTION
**Pull request by Seluj78**

### CODE REVIEW

**Comment by JulienPalard** [comment]
LGTM

File: Doc/library/idle.rst [hunk 1]
```diff
@@ -689,7 +689,7 @@ When IDLE first starts, it attempts to read user configuration files in
 ``~/.idlerc/`` (~ is one's home directory).  If there is a problem, an error
 message should be displayed.  Leaving aside random disk glitches, this can
 be prevented by never editing the files by hand, using the configuration
```
Comment (akuchling) [inline Doc/library/idle.rst]: I think this change still leaves the grammar of the sentence mangled. It should be something like "by never editing the files by hand, instead using the configuration dialog under Options."

**Review by akuchling — DISMISSED** [review]

File: Doc/library/idle.rst (hunk 1 above)
Comment (terryjreedy) [reply Doc/library/idle.rst]: Thank you for seeing what I was trying to say.  I made this change, with '.' instead of ','.

**Review by terryjreedy — APPROVED** [review]

**Comment by terryjreedy** [comment]
Jules, thank you for this PR reporting my incomplete editing.  After incorporating Andrew's suggestion, I also improved a bit the paragraphs before and after.

**Comment by terryjreedy** [comment]
Backports are merged.

**Pull request merged**